### PR TITLE
Jj/lint errors

### PR DESCRIPTION
**Pull request by JuliaJakubcova**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -22,16 +22,17 @@ subprojects {
         version.set("0.37.2")
         enableExperimentalRules.set(true)
         verbose.set(true)
+        debug.set(true)
```
Comment (samhill303) [inline build.gradle.kts:25]: Should we be leaving debug on?

File: shared-ui/src/androidMain/kotlin/co/touchlab/droidcon/ui/util/LocalImage.jvm.kt [hunk 2]
```diff
@@ -19,6 +20,7 @@ import androidx.compose.ui.res.painterResource
 import co.touchlab.droidcon.ui.icons.Warning
 import co.touchlab.droidcon.ui.theme.Dimensions
 
+@SuppressLint("ComposableNaming", "DiscouragedApi")
```
Comment (samhill303) [inline shared-ui/src/androidMain/kotlin/co/touchlab/droidcon/ui/util/LocalImage.jvm.kt:23]: Comment on discouragedAPI, I think naming is obvious. (Is the non-standard naming required?)

File: shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/util/NavigationController.kt [hunk 3]
```diff
@@ -130,20 +128,20 @@ class NavigationController : BaseViewModel() {
...
-            Surface {
+            Surface(modifier = itemModifier) {
                 item.content(it)
             }
         }
     }
 
     @Composable
     internal fun <T : Any> Pushed(item: MutableObservableProperty<T?>, content: @Composable (T) -> Unit) {
-        val refTracking = remember {
```
Comment (samhill303) [inline shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/util/NavigationController.kt:146]: the name refTracking makes me wonder if it was intentionally assigning to a variable even though it wasn't used. I still imagine it isn't needed, but maybe double check

**Review by samhill303 — COMMENTED** [review]
only got through a little more than half the files, will followup

File: shared/src/iosMain/kotlin/co/touchlab/droidcon/Koin.ios.kt [hunk 4]
```diff
@@ -22,12 +22,12 @@ actual val platformModule = module {
     single<SqlDriver> { SqlDelightDriverFactory().createDriver() }
 
     single<HttpClientEngine> {
-        Ios.create {}
+        Darwin.create {}
```
Comment (samhill303) [inline shared/src/iosMain/kotlin/co/touchlab/droidcon/Koin.ios.kt:25]: Switching ktor engines seems significant. What prompted this?

Comment (samhill303) [reply shared/src/iosMain/kotlin/co/touchlab/droidcon/Koin.ios.kt:25]: I see it's just an alias that is depricated

File: build.gradle.kts (hunk 1 above)
Comment (JuliaJakubcova) [reply build.gradle.kts:25]: Seems it's not needed, I'll remove this line.

File: shared-ui/src/androidMain/kotlin/co/touchlab/droidcon/ui/util/LocalImage.jvm.kt (hunk 2 above)
Comment (JuliaJakubcova) [reply shared-ui/src/androidMain/kotlin/co/touchlab/droidcon/ui/util/LocalImage.jvm.kt:23]: Tadeas came up with this name, it's just for internal use, the developer uses `LocalImage` which has the default values set. We can change the name of `__LocalImage` if you have an idea for a better name :)

File: shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/util/NavigationController.kt (hunk 3 above)
Comment (JuliaJakubcova) [reply shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/util/NavigationController.kt:146]: I tested the navigation and it works correctly, I think.

**Review by samhill303 — APPROVED** [review]

**Pull request merged**